WooCommerce/Classes/ViewRelated/Dashboard/JetpackInstall/JetpackInstallSteps.swift

```diff
@@ -39,10 +39,51 @@ extension JetpackInstallStep {
         }
     }
 
+    /// Error message to display when Jetpack install fails on the step
+    ///
+    var errorMessage: String? {
+        switch self {
+        case .installation:
+            return Localization.installErrorMessage
+        case .activation:
+            return Localization.activationErrorMessage
+        case .connection:
+            return Localization.connectionErrorMessage
+        case .done:
+            return nil
+        }
+    }
+
+    /// Title of the CTA to display when Jetpack install fails on the step
+    ///
+    var errorActionTitle: String? {
+        switch self {
+        case .installation:
+            return Localization.wpAdminInstallAction
+        case .activation:
+            return Localization.wpAdminActivateAction
+        case .connection:
+            return Localization.checkConnectionAction
+        case .done:
+            return nil
+        }
+    }
+
     private enum Localization {
         static let installationStep = NSLocalizedString("Installing Jetpack", comment: "Name of installing Jetpack plugin step")
         static let activationStep = NSLocalizedString("Activating", comment: "Name of the activation Jetpack plugin step")
         static let connectionStep = NSLocalizedString("Connecting your store", comment: "Name of the step to connect the store to Jetpack")
         static let finalStep = NSLocalizedString("All done", comment: "Name of final step in Install Jetpack flow.")
+        static let installErrorMessage = NSLocalizedString("Please try again. Alternatively, you can install Jetpack through your WP-Admin.",
+                                                    comment: "Error message when Jetpack install fails")
+        static let activationErrorMessage = NSLocalizedString("Please try again. Alternatively, you can activate Jetpack through your WP-Admin.",
```

Comment:
Is it correct to have the same error message for these two cases?

Reply:
The messages are very similar and only different in 2 words "install" and "activate" - I understand that it can be easy to be overlooked 😄